plugins/modules/uptime_check.py

```diff
@@ -61,11 +103,18 @@ def main():
         ## Set input variables
         fields = {
                 "apikey": {"type": "str", "required": True, "no_log": True},
-                "url": {"type": "str", "required": True},
+                "host": {"type": "str", "required": True},

```

Comment:
The types here should match what is in the documentation above, which will mean converting all the non-string types into strings before calling pingdompy. Worth checking first what the api requires, as these might be easier to specify these as strings.